src/main/java/net/sf/log4jdbc/StatementSpy.java

```diff
@@ -25,15 +25,34 @@
 import java.util.List;
 import java.util.ArrayList;
 
+import static net.sf.log4jdbc.Utilities.getConfigProp;
+
 /**
  * Wraps a Statement and reports method calls, returns and exceptions.
  *
  * jdbc 4 version
  *
  * @author Arthur Blake
  */
-public class StatementSpy implements Statement, Spy
-{
+public class StatementSpy implements Statement, Spy {
+  static final String SQL_SHOW_PARAMS_KEY = "LOG4JDBC_SQL_SHOW_PARAMS";
+
+  static final boolean showParams;
+
+  static {
+    showParams = getConfigProp(SQL_SHOW_PARAMS_KEY, v -> {
+      switch (v.trim().toLowerCase()) {
+        case "true":
+          return true;
+        case "false":
+          return false;
+        default:
+          throw new IllegalArgumentException(String.format(

```

Comment:
if we don't pass value for the this flag does jenkins defaults to false? @vivekragunathan @davidcheney-livongo